pkgs/by-name/ra/raycast/package.nix

```diff
@@ -3,20 +3,29 @@
   stdenvNoCC,
   fetchurl,
   writeShellApplication,
+  cacert,
   curl,
   jq,
-  common-updater-scripts,
+  nix,
   undmg,
 }:
 
+let
+  hashes = {
+    arm = "sha256-q3pX/mOl/u9KMcAfvXm4giYKjnTB903N1ibubvaO9Uw=";
+    x86_64 = "sha256-l61AVKx+aYmgnVK8d+by2pKiu1cIAueLipRjOzCvib4=";
+  };
+in
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "raycast";
-  version = "1.88.3";
+  version = "1.88.4";
+  isArm = stdenvNoCC.hostPlatform.isDarwin || stdenvNoCC.hostPlatform.isAarch64;
+  archBuild = if finalAttrs.isArm then "arm" else "x86_64";
```

Comment:
`isArm` and `archBuild` are not arguments of `mkDerivation`

IMO it'd be better to put them above in the let-in binding or do smth similar to this:
https://github.com/NixOS/nixpkgs/blob/c9f456aaad5888b0e7745f0bbe91ab1b73fb838f/pkgs/by-name/os/osu-lazer-bin/package.nix#L15-L32

Reply:
You're right this is better!